api/core/domain/models/_mistral.py

```diff
@@ -234,7 +234,29 @@ def mistral_models() -> dict[Model, ModelData | LatestModel | DeprecatedModel]:
             provider_for_pricing=Provider.MISTRAL_AI,
             icon_url="https://workflowai.blob.core.windows.net/workflowai-public/mistral.svg",
             release_date=date(2024, 7, 24),
-            quality_index=481,  # MMLU=63.47, GPQA=38.35
+            quality_data=QualityData(mmlu=63.47, gpqa=38.35),
+            provider_name=DisplayedProvider.MISTRAL_AI.value,
+            supports_tool_calling=True,
+        ),
+        Model.MISTRAL_MEDIUM_2505: ModelData(
```

Comment:
Mistral Medium 3 ?

Reply:
The ID does not contain the `3` it's weird.
All the other models have a number, but it's never in the ID or the doc. I'll change the name to 3 here